<?xml version="1.0" encoding="UTF-8"?>

<!---
File:        new_device_dialog.ui                                
Case:        VUT, FIT, ICP, project                                      
Author:      Vladislav Sokolovskii, xsokol15@stud.fit.vutbr.cz        
Date:        summer semester 2021
Brief:       Design of the New Device dialog.      
-->

<ui version="4.0">
 <class>NewDeviceDialog</class>
 <widget class="QDialog" name="NewDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New device</string>
  </property>
  <widget class="QWidget" name="verticalWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="device_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Device type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="device_type">
        <item>
         <property name="text">
          <string>Light</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thermometer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hygrometer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wattmeter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Move sensor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Camera</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relay</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Valve</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thermostat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lock</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Topic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="device_topic"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ok">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
